Gemfile

```diff
@@ -56,6 +56,7 @@ gem "manageiq-password",              "~>0.3",         :require => false
 gem "manageiq-postgres_ha_admin",     "~>3.1",         :require => false
 gem "memoist",                        "~>0.15.0",      :require => false
 gem "mime-types",                     "~>3.0",         :path => File.expand_path("mime-types-redirector", __dir__)
+gem "money",                          "~>6.13.5",      :require => false
```

Comment:
@Fryguy I think the `Money` class was coming from the `money-rails` gem here - https://github.com/ManageIQ/manageiq-consumption/blob/c7198b50c03944497b00fd28284caf723a88ff1b/manageiq-consumption.gemspec#L20, not the `money` gem. I'm not sure that it matters since we only use the gem for its list of currencies, but I thought I should mention it. I had added the `money-rails` gem to the [PR](https://github.com/ManageIQ/manageiq/pull/19550/files) that decouples us from `manageiq-consumption` and noticed this what I tried to rebase it with this change.